tests/flags.rs

```diff
@@ -87,6 +87,24 @@ fn flag_using_long() {
     assert!(m.is_present("color"));
 }
 
+#[cfg(feature = "suggestions")]
+#[test]
+fn flag_using_long_with_literals() {
+    use clap::ErrorKind;
+
+    let m = App::new("flag")
+        .args(&[Arg::from("--rainbow 'yet another flag'").multiple_occurrences(true)])
```

Comment:
Can you please use `Arg::new` instead of `Arg::from` and `arg` instead of `args`? And let's remove `multiple_occurrences`.

Reply:
Done.